spec/tools/environment_builders/openstack/services/compute/builder.rb

```diff
@@ -64,9 +64,13 @@ def find_or_create_servers(volumes, networks, security_groups, images, networkin
                             }) if volume
             end
 
-            if (network_name = server.delete(:__network_name))
-              network = networks.detect { |x| x.name == network_name }
-              server.merge!(:nics => [{"net_id" => network.id}]) if network
+            if (network_names = server.delete(:__network_names))
+              nics = []
+              network_names.each do |network_name|
+                network = networks.detect { |x| x.name == network_name }
+                nics << {"net_id" => network.id} if network
+              end
```

Comment:
@DNNX hehe, yeah probably no need for that much optimization, this code is outside of production use, it's used for building VCR envs, that are committed to code and used in CI for testing